### PR TITLE
Define publicKeyJwk

### PR DESCRIPTION
**Pull request by OR13**
create `security-v3-unstable.jsonld`

add definition for `publicKeyJwk`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1447,6 +1447,39 @@ <h3>publicKeyBase58</h3>
...
+          <dt>Domain</dt>
+          <dd>Key</dd>
+          <dt>Range</dt>
+          <dd>xsd:string</dd>
+        </dl>
+
+        <pre class="example prettyprint language-jsonld">
+{
+  "id": "did:example:123#_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A",
+  "type": "JsonWebKey2020",
```
Comment (msporny) [inline index.html:1470]: Blocked by discussion about JsonWebKey2020/JwkVerificationKey2020... we need to resolve this to pull in this example.

**Comment by OR13** [comment]
@msporny @kdenhartog I believe we may have a special topic call which would allow for us to address this in the DID WG... its the same topic call that will address: 

- https://github.com/w3c/did-core/issues/185

**Comment by kdenhartog** [comment]
> 
> 
> @msporny @kdenhartog I believe we may have a special topic call which would allow for us to address this in the DID WG... its the same topic call that will address:
> 
>     * [w3c/did-core#185](https://github.com/w3c/did-core/issues/185)

I haven't seen a topic call for that come up yet, but may have missed it. Do you know what day it is or is it still TBD and I should keep my eye out for it?

**Comment by OR13** [comment]
Just putting this out there: https://docs.microsoft.com/en-us/microsoft-edge/dev-guide/windows-integration/web-authentication

```
attestationObject = cbor.decodeFirstSync(Buffer.from(attestation.attestationObject, 'base64'));
    authenticatorData = parseAuthenticatorData(attestationObject.authData);
    var credential = await storage.Credentials.create({
        id: authenticatorData.attestedCredentialData.credentialId.toString('base64'),
        publicKeyJwk: authenticatorData.attestedCredentialData.publicKeyJwk,
        signCount: authenticatorData.signCount
    });
```

**Comment by OR13** [comment]
@msporny what can I do to unblock this?

**Comment by peacekeeper** [comment]
Related to the `transform-keys` DID parameter: https://w3c.github.io/did-spec-registries/#transform-keys-0

**Comment by OR13** [comment]
@msporny @dlongley I believe we settled this. note that this is still the "unstable context"... please review again.

File: index.html (hunk 1 above)
Comment (OR13) [reply index.html:1656]: Will update the name, if the name changes... all of the examples in this repo need a refresh.

**Comment by OR13** [comment]
@msporny as you are currently the only code owner, i am awaiting your review.

**Pull request merged**